rifServices/src/main/java/org/sahsu/rif/services/rest/WebService.java

```diff
@@ -1388,10 +1388,68 @@ private RIFStudySubmission getRIFSubmissionFromJSONSource(
 			JSONObject rifJobSubmission = jsonObject.optJSONObject("rif_job_submission");
 			if (rifJobSubmission != null) {
 				JSONObject studySelection = rifJobSubmission.optJSONObject("study_selection");
+				JSONObject disease_mapping_study = rifJobSubmission.optJSONObject("disease_mapping_study");
```

Comment:
`disease_mapping_study` and the others with underscores are non-standard Java names. And since only one or the other can exist, why not just have a single one, called `study`?

Reply:
Yes, good idea